manual/source/datarules.rst

```diff
@@ -643,111 +643,119 @@ section, which describes the routines available to program these
+#. Pick the :index:`!default` :ref:`NXentry` group.
 
-   If the *root* has an attribute ``default``, then its value
-   is the name of the ``NXentry`` group to be used.  Otherwise,
-   pick any ``NXentry`` group.  This is trivial if there is only one 
-   ``NXentry`` group.
+   If the *root* has an attribute ``default``, then its value is the name of the
+   ``NXentry`` group to be used.  (The value of the attribute ``default`` must
+   be the name of a child of the current group and that child must be a NeXus group or link
+   to a NeXus group. The value must match the :ref:`regular expression pattern
+   for NXDL group and field names <validItemName>`.) Otherwise, pick any
+   ``NXentry`` group.  This is trivial if there is only one ``NXentry`` group.
 
    .. compound::
-   
+
        .. _fig.flowchart-NXroot-default:
-   
+
        .. figure:: img/flowchart-NXroot-default.png
            :alt: fig.flowchart-NXroot-default
            :width: 60%
-   
+
            Find plottable data: select the ``NXentry`` group
 
 #. Pick the default :ref:`NXdata` group.
 
-   Open the ``NXentry`` group selected above.
-   If it has an attribute ``default``, then its value
-   is the name of the ``NXdata`` group to be used.  Otherwise,
-   pick any ``NXdata`` group.  This is trivial if there is only one 
-   ``NXdata`` group.
+   Open the ``NXentry`` group selected above. If it has an attribute
+   ``default``, then its value is the name of the ``NXdata`` group to be used.
+   (The value of the attribute ``default`` must be the name of a child of the current group
+   and that child must be a NeXus group or link to a NeXus group. The value must
+   match the :ref:`regular expression pattern for NXDL group and field names
+   <validItemName>`.) Otherwise, pick any ``NXdata`` group. This is trivial if
+   there is only one ``NXdata`` group.
 
    .. compound::
-   
+
        .. _fig.flowchart-NXentry-default:
-   
+
        .. figure:: img/flowchart-NXentry-default.png
            :alt: fig.flowchart-NXentry-default
            :width: 60%
-   
+
            Find plottable data: select the ``NXdata`` group
 
 .. index:: signal data
 
 #. Pick the default plottable field (the *signal* data).
 
-   Open the ``NXdata`` group selected above.
-   If it has an attribute ``signal``, then its value
-   is the name of the field (dataset) to be plotted.
-   If no ``signal`` attribute is present on the
-   ``NXdata`` group, then proceed to try an
-   :ref:`older NeXus method<Find-Plottable-Data-v2>`
-   to find the default plottable data.
+   Open the ``NXdata`` group selected above. If it has an attribute ``signal``,
+   then its value is the name of the field (dataset) to be plotted. (The value
+   of the attribute ``signal`` must be the name of a child of the current group and that
+   child must be a NeXus field or link to a NeXus field. The value must match
+   the :ref:`regular expression pattern for NXDL group and field names
```

Comment:
again, this is not the right place to discuss the naming scheme.

Reply:
In fact this whole section ignores the existence of NXsubentry. Perhaps this should be split off into a separate issue?

Reply:
Reference to the naming scheme emphasizes that the name used is [not a path through sublevel(s)](https://github.com/nexusformat/definitions/issues/1063) but an immediate child of this group.  But the text using this reference can be softened.  I do not understand why *should* is a replacement for *must* in this case.  Either it names an existing child group or it is not used.